backend/api/professor/cohorts.py

```
@@ -112,6 +129,62 @@ async def get_cohorts(
+                                    instance.completion_percentage = (completed_scenes / total_scenes) * 100.0
+                    if not end_dt:
+                        end_dt = datetime.now(timezone.utc)
+                    if start_dt:
+                        if start_dt.tzinfo is None:
+                            start_dt = start_dt.replace(tzinfo=timezone.utc)
+                        if end_dt and end_dt.tzinfo is None:
+                            end_dt = end_dt.replace(tzinfo=timezone.utc)
+                        delta = end_dt - start_dt
+                        instance.total_time_spent = max(0, int(delta.total_seconds()))
+                db.commit()
+        return {"status": "ok", "refreshed": True}
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)